### PR TITLE
Update plot_examples.py

### PR DESCRIPTION
**Pull request by affaannn**
#### Brief description of what is fixed or changed

Declared symbolic variables `x`,`y`,`u` and `v` in sigle line instead of four.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: examples/beginner/plot_examples.py [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
                             plot3d)
 
 lx = range(5)
 ly = [i**2 for i in lx]
 
-x = Symbol('x')
-y = Symbol('y')
-u = Symbol('u')
-v = Symbol('v')
+x, y, u, v = Symbols('x,y,u,v')
```
Comment (1e9abhi1e10) [inline examples/beginner/plot_examples.py:12]: Should be a space after comma.

Comment (1e9abhi1e10) [inline examples/beginner/plot_examples.py:12]: This is not the right way to use symbols 
Should be like this 
`x, y, u, v = symbols('x y u v')`

File: examples/beginner/plot_examples.py [hunk 2]
```diff
@@ -1,18 +1,15 @@
 #! /usr/bin/env python
 # Check the plot docstring
 
-from sympy import Symbol, exp, sin, cos
+from sympy import Symbols, exp, sin, cos
```
Comment (1e9abhi1e10) [inline examples/beginner/plot_examples.py:4]: it should be `symbols`

**Comment by 1e9abhi1e10** [comment]
Look like it is your first pull request, for that you need to add your name and email in the `.mailmap` file 
read [here](https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file) for instructions.

File: examples/beginner/plot_examples.py (hunk 1 above)
Comment (affaannn) [reply examples/beginner/plot_examples.py]: thank you for your reviews
I changed it and committed

**Comment by affaannn** [comment]
> Look like it is your first pull request, for that you need to add your name and email in the `.mailmap` file read [here](https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file) for instructions.

can  I simply update my name and mail in .mailmap then commit it?

**Comment by hanspi42** [comment]
> can I simply update my name and mail in .mailmap then commit it?

You should read the description at the start of the `.mailmap` file, it tells you how to do it.

**Comment by hanspi42** [comment]
By the way, this change does not need to be in the release notes, so I change it.

**Comment by sylee957** [comment]
I don't think that this is a necessary change though. 
Although the `symbols` are convenient, it is trivial to write four instead of one.
And now I don't always prefer `symbols` because it doesn't statically gurantee that there are 4 elements, which are always in the `Symbol` class by type checking.

**Pull request closed without merging**